src/features/apiDelegation/components/OverviewPageContent/OverviewPageContent.tsx

```diff
@@ -178,82 +178,81 @@ export const OverviewPageContent = ({
   };
 
   return (
-    <div className={common.pageContent}>
-      <div className={classes.overviewActionBarContainer}>
-        {!isSm && <h2 className={classes.pageContentText}>{overviewText}</h2>}
-        {layout === LayoutState.Offered && (
-          <div className={classes.delegateNewButton}>
-            <Button
-              variant={ButtonVariant.Outline}
-              onClick={goToStartDelegation}
-              icon={<Add />}
-              fullWidth={isSm}
-            >
-              {t('api_delegation.delegate_new_org')}
-            </Button>
-          </div>
-        )}
-        <Panel
-          title={t('api_delegation.card_title')}
-          forceMobileLayout={isSm}
-          showIcon={!isSm}
-        >
-          {t('api_delegation.api_panel_content')}{' '}
-          <a
-            className={classes.link}
-            href='https://samarbeid.digdir.no/maskinporten/maskinporten/25'
-            target='_blank'
-            rel='noreferrer'
+    <div className={classes.overviewActionBarContainer}>
+      {!isSm && <h2 className={classes.pageContentText}>{overviewText}</h2>}
+      {layout === LayoutState.Offered && (
+        <div className={classes.delegateNewButton}>
+          <Button
+            variant={ButtonVariant.Outline}
+            onClick={goToStartDelegation}
+            icon={<Add />}
+            fullWidth={isSm}
+            size='medium'
           >
-            {t('common.maskinporten')}
-          </a>
-        </Panel>
-        <div className={classes.explanatoryContainer}>
-          {overviewOrgs.length > 0 && (
-            <>
-              {isSm ? (
-                <h3 className={classes.apiSubheading}>{accessesHeader}</h3>
+            {t('api_delegation.delegate_new_org')}
```

Comment:
Need a little bit of info what these changes are about

Reply:
DS updated their buttons to use new sizes, so I had to specify medium size on them to keep about the same size as they used to. That, and I removed an outer div that wasn't being used for anything anymore so the indents changed.
That's why git marks everything as changed 😅